Src/BootCamp.Chapter/Conversions.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace BootCamp.Chapter
+{
+    class Conversions
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Convert Decimal To Binary");
+            ConvertDecimalToBinary();
+            Console.WriteLine();
+            Console.WriteLine("Convert Binary To Decimal");
+            ConvertBinaryToDecimal();
+        }
+
+        private static void ConvertDecimalToBinary()
+        {
+            int i;
+            int[] binaryHolder = new int[40];
```

Comment:
Why 40? It's a magic number, please explain why 40, preferably use named constant.

Reply:
I'm creating an array to hold the "string" of bits that represent the decimal number. I don't know how to do this without pre-naming the length of the array. 40 is an arbitrary number, though. It seems like there must be a better, more dynamic way. I don't know what it is.

I think "magic number" is whenever I'm defining a static value in my algorithm, I should define it as a constant at the top of the code so it is more easily found and edited. Thanks for that. I'll make changes wherever I have magic numbers.